napari/utils/transforms/_tests/test_transforms.py

```diff
@@ -1,12 +1,18 @@
 import numpy as np
 import numpy.testing as npt
+import pint
 import pytest
 from scipy.stats import special_ortho_group
 
 from napari.utils.transforms import Affine, CompositeAffine, ScaleTranslate
 
 transform_types = [Affine, CompositeAffine, ScaleTranslate]
 
+affine_type = [Affine, CompositeAffine]
+
+REG = pint.get_application_registry()
+PIXEL = REG.pixel
```

Comment:
Wait is pixel a built-in unit?? Since when??? (I have a memory of having to manually define it...)

Reply:
I do not know. But it is builtin. 
https://github.com/hgrecco/pint/blob/master/pint/default_en.txt#L671

Reply:
From git-blame it seems ancient (8+ years). I must have missed it somehow. 😅 Or I'm thinking of a different unit... Anyway, cool!